src/test/java/org/xwiki/git/internal/DefaultGitManagerTest.java

```diff
@@ -109,6 +111,23 @@ public void getRepositoryWhenWrongCredentialsProvided() throws Exception
         }
     }
 
+    @Test
+    public void getRepositoryBareWithCredentials() throws Exception
```

Comment:
There's no need to test with credentials again since it's been tested already in another test.

Reply:
Updated.